README.md

```diff
@@ -1 +1,51 @@
 **This is not an officially supported Google product.**
+
+# Apache Ranger Gcs Plugin
+
+Currently, when users request access to objects stored on gcs, gcs connector uses service accounts instead of user accounts to access gcs.
+For example, when Bob wants to read a file stored on gcs, gcs connector uses the service account on that vm to fetch the file.
+Object accesses will always succeed no matter whether the actual users have permission or not.
```

Comment:
Let's not talk about this. Just the second graph (add mentioning of GCS) should be fine.